_articles/de/maintaining-balance-for-open-source-maintainers.md

```diff
@@ -218,4 +218,5 @@ This guide was written by [@abbycabs](https://github.com/abbycabs) with contribu
 [@thisisnic](https://github.com/thisisnic)
 [@tudoramariei](https://github.com/tudoramariei)
 [@UlisesGascon](https://github.com/UlisesGascon)
-[@waldyrious](https://github.com/waldyrious) + many others!
+[@waldyrious](https://github.com/waldyrious)
+[@WebSnke(https://github.com/WebSnke)] + vielen anderen!
```

Comment:
This link is malformed and is causing a test failure! Please fix and we will merge.
```suggestion
[@WebSnke](https://github.com/WebSnke) + vielen anderen!
```